packages/devtools_app/lib/src/theme.dart

```diff
@@ -226,36 +223,53 @@ extension DevToolsColorScheme on ColorScheme {
       isLight ? Colors.grey[300] : Colors.grey[700];
 
   // Title of the hover card.
-  TextStyle get hoverTitleTextStyle => const TextStyle(
-        color: Colors.black,
-        fontWeight: FontWeight.bold,
-        fontSize: 16,
+  TextStyle get hoverTitleTextStyle => TextStyle(
+        color: isLight ? Colors.black87 : Colors.white54,
```

Comment:
what about `defaultForeground` or `contrastForeground` from ui/theme.dart? While you're modifying these files, let's pull in the getters from the extension in ui/theme.dart to this extension. Then we can delete the ui/theme.dart file so we have all these extension methods in one place.

Reply:
Used defaultForeground for titles and contrastForeground for values looks nice and removed ui/theme.dart and moved to theme.dart.